resources/views/forum/topics/logs/show.blade.php

```diff
@@ -0,0 +1,59 @@
+{{--
+    Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+    See the LICENCE file in the repository root for full licence text.
+--}}
+@extends('master', [
+    'titlePrepend' => trans('forum.topic.logs.title'),
+])
+
+@section('content')
+    @include('forum.topics._floating_header')
+    @include('forum._header', [
+        'isTopicLogs' => true,
+        'forum' => $topic->forum,
+        'topic' => $topic,
+    ])
+
+    <div class="osu-page osu-page--generic osu-page--full">
+        @if ($logs->count() > 0)
+            <table class="forum-topic-logs-table table">
```

Comment:
bootstrap table isn't all that useful here (and using bootstrap components should be generally avoided)